docs/05-10-api.md

```diff
@@ -1,25 +1,62 @@
 # Watcher CR
 
-The Watcher CR configures Kyma Control Plane (KCP) setup and Runtime Watcher on Kyma runtimes. Read the document to see the the parameters the CR consists of. **spec.labelsToWatch** Configures the validation webhook configuration in a Kyma runtime (SKR)
+The [Watcher CR](https://github.com/kyma-project/lifecycle-manager/blob/main/api/v1beta2/watcher_types.go#L121) configures Kyma Control Plane (KCP) setup and Runtime Watcher on Kyma runtimes. Read the document to see the the parameters the CR consists of.
 
-## **spec.labelsToWatch**
+## Runtime-Watcher configuration
 
-The resources you're watching need to have the `operator.kyma-project.io/watched-by` label. The label also informs where to route requests to.
+### **spec.resourceToWatch**
 
-## **spec.resourceToWatch**
+Defines the resources to watch by Runtime Watcher. By default, the **version** value is a wildcard (`*`) so an update is not required after the API version of a kind changes.
```

Comment:
```suggestion
Defines which resources the Runtime Watcher watches. By default, the **version** value is a wildcard (`*`), so an update is not required after the API version changes.
```

If we can drop the "of a kind" without losing meaning, it would be much easier to read.